src/test/java/com/google/devtools/build/lib/remote/BuildWithoutTheBytesIntegrationTestBase.java

```diff
@@ -1055,6 +1097,83 @@ public void downloadToplevel_unresolvedSymlink() throws Exception {
     assertSymlink("foo-link", PathFragment.create("/some/path"));
   }
 
+  @Test
+  public void downloadMinimal_fileWrite(
```

Comment:
Can you also parameterize these tests for `ctx.actions.write_file` and `ctx.actions.expand_template`? (It's fine to use a dummy template with no substitutions, I'm only concerned about covering the separate code path in `LocalTemplateExpansionStrategy`)